Plugins/Public/base_plugin/Main.cpp

```diff
@@ -90,8 +90,19 @@ uint set_tick_time = 16;
 // How much damage do we heal per repair cycle?
 uint repair_per_repair_cycle = 60000;
 
-/// If the shield is up then damage to the base is changed by this multiplier.
-float set_shield_damage_multiplier = 0.03f;
+
+// set of configurable variables defining the diminishing returns on damage during POB siege
+// POB starts at base_shield_strength, then every 'threshold' of damage taken, 
+// shield goes up in absorption by the 'increment'
```

Comment:
does not explain 'flat' vs. 'percent' distinction

Reply:
Wanted an ability to customize the curve of POB toughness to core level.
I suppose I could just make a map where we strictly define threshold size from a map<coreLvl, threshold>?

Reply:
Update: I have implemented the map of corelvl->shield reinforcement threshold.

Reply:
... I don't really feel like you've explained anything here but I suppose I should consider it separately with new code